addons/openebs/enable

```diff
@@ -30,12 +66,14 @@ fi
 # make sure the "openebs" namespace exist
 $KUBECTL create namespace "$OPENEBS_NS" > /dev/null 2>&1 || true
 
+
 $HELM repo add openebs https://openebs.github.io/charts
 $HELM repo update
 $HELM -n openebs install openebs openebs/openebs \
-    --version 3.0.x \
+    --version ${OPENEBS_VERSION} \
     --set cstor.enabled=true \
     --set jiva.enabled=true \
+    --set jiva.replicas=${JIVA_REPLICAS} \
```

Comment:
@zacbayhan Shouldn't this instead be `--set jiva.defaultPolicy.replicas=${JIVA_REPLICAS}`? Looking at https://github.com/openebs/jiva-operator/blob/develop/deploy/helm/charts/templates/default-policy.yaml#L9

Reply:
I am investigating that now, one a cluster that is running which creates the correct number of replicas it still shows that the jiva.defaultPolicy.replicas=3 but spins up the number of replicas set via `jiva.replicas=${JIVA_REPLICAS}`

Reply:
`jiva.replicas`, is a valid configuration and is in the top layer of the values file, https://github.com/openebs/openebs/blob/main/k8s/charts/openebs/values.yaml#L134-L137

I have observed the desired behavior by setting that however it doesn't impact the defaultPolicy which still indicates a desired replication factor of 3.

Reply:
Ok, so I have updated the pr to include the changes via the `jiva.defaultPolicy.replicas` setting vs the `jiva.replicas` settings. While it looks like the 2nd still produces the desired outcome it doesn't update the crd.